charts/zalenium/values.yaml

```diff
@@ -89,18 +89,18 @@ hub:
 
   ## Environment variables passed to Zalenium hub.
   ## https://github.com/zalando/zalenium/blob/master/docs/usage_examples.md
-  desiredContainers: 2
-  maxDockerSeleniumContainers: 10
+  desiredContainers: 4
+  maxDockerSeleniumContainers: 30
   videoRecordingEnabled: true
   # the limits/requests below are for Selenium pods (containers)
   cpuRequest: 250m
   cpuLimit: 1000m
   memRequest: 500Mi
   memLimit: 2Gi
-  screenWidth: 1440
-  screenHeight: 900
-  timeZone: "UTC"
-  seleniumImageName: "elgalu/selenium"
+  screenWidth: 1280
+  screenHeight: 720
+  timeZone: "UTC-2"
+  seleniumImageName: "colaslorenzo/docker-selenium:ff_102_driver_29_chr_103"
```

Comment:
Is the way to create this container image documented in some way?

Reply:
Point 2 here: https://nexplore.atlassian.net/wiki/spaces/GQ/pages/2343403870/Zalenium+installation+and+configuration+LOCAL